contracts/exchange-libs/src/reference_functions.ts

```diff
@@ -113,3 +113,30 @@ export function calculateFillResults(
         protocolFeePaid: safeMul(protocolFeeMultiplier, gasPrice),
     };
 }
+
+export const LibFractions = {
```

Comment:
I feel like @dorothy-zbornak added a math library to `contracts-utils` or `contracts-test-utils` that we could merge this into.